index.html

```diff
@@ -1083,6 +1090,82 @@ <h3>Multikey</h3>
 
           </section>
 
+          <section>
+            <h3>JsonWebKey</h3>
+            <p>
+The JSON Web Key data model is a specific type of <a>verification method</a>
+that utilizes the JWK specification [[RFC7517]] to encode key types into a
+set of parameters. To encode a JSON Web Key, the <a>verification method</a>
+`type` MUST be set to `JsonWebKey` and the `publicKeyJwk` value MUST be a
+valid JWK value as described in [[RFC7517]]. An example of a JSON Web Key is
+provided below:
+            </p>
+
+            <pre class="example nohighlight"
+              title="JSON Web Key encoding of a Ed25519 public key">
+{
+  "@context": ["https://www.w3.org/ns/security/jwk/v1"],
+  "id": "did:example:123456789abcdefghi#key-1",
+  "type": "JsonWebKey",
+  "controller": "did:example:123456789abcdefghi",
+  "publicKeyJwk": {
+    "kty": "OKP",
+    "alg": "EdDSA"
+    "crv": "Ed25519",
+    "kid": "key-1",
+    "x": "_1EiHquO2aUx9JARSu0P8jdYT_OVneYxYOnOMAmUcFI",
+  }
+}
+            </pre>
+
+            <p>
+In the example above, the `publicKeyJwk` value contains the JSON Web Key.
+The `kty` property encodes the key type of "OKP", which means
+"Octet string key pairs". The `alg` property identifies the algorithm intended
+for use with the public key. The `crv` property identifies the particular
+curve type of the public key. The `kid` property specifies how the public key
+might be referenced in software systems; if present, the value SHOULD match the
+`id` property of the encapsulating `JsonWebKey` object. Finally, the `x`
+property specifies the point on the Ed25519 curve that is associated with the
+public key.
+            </p>
+
+            <p>
+The `publicKeyJwk` property MUST NOT contain any property marked as
+"Private" in any registry contained in the JOSE Registries [[JOSE-REGISTRIES]].
+            </p>
+
+            <p>
+The JSON Web Key data model is also capable of encoding secret keys, sometimes
+referred to as <em>private keys</em>.
```

Comment:
Integrating [the suggestion](https://github.com/w3c/vc-data-integrity/pull/135/files#r1284562689) from @OR13 --
```suggestion
The JSON Web Key data model is also capable of encoding private information,
sometimes referred to as <em>private keys</em> or <em>secret keys</em>.
```

Reply:
^ I support the language above, but in case we want to be even more precise, "private information" is not "secrete keys".... its parameters of secret keys... for example:

https://www.iana.org/assignments/jose/jose.xhtml#web-key-set-parameters

![Screen Shot 2023-08-04 at 5 10 31 PM](https://github.com/w3c/vc-data-integrity/assets/8295856/2098cfdb-75d6-4196-be98-c26a3016ec5c)

Reply:
Updated language. 

> `The JSON Web Key data model is also capable of encoding private information,
including <em>private keys</em> and/or <em>secret keys</em>.`

Reply:
> I support the language above, but in case we want to be even more precise, "private information" is not "secrete keys" or "private keys".... its parameters of "secrete keys" or "private keys"... for example:

Yes, and I believe this is one of the challenges with some key formats. I think it's better for app developers to be able to treat key values as opaque and completely omit secret keys without having to know the details around validating and parsing them to look for specific internal components, etc. This is best left to another layer.

Reply:
> The advice given in data integrity regarding "d" does not actually protect people... since RSA does not use "d"... and lots of people still use RSA saddly.

This might have been a typo; RSA does use "d" -- that's where it came from originally.

Reply:
Yes, I meant to say... omitting ONLY `d` (which is the current guidance) is bad advice... 

Just like it would be bad advice to say that only uranium is radioactive... There are other radioactive elements... and there are other "private information"  parameters that should not be seen in public keys.

Reply:
> Yes, I meant to say... omitting ONLY `d` (which is the current guidance) is bad advice...

Agreed -- it's good as an example but that example should say that it's insufficient to just omit that one thing and that `<reference X>` must be checked and every possible private or secret (whichever terminology is appropriate) component listed similarly omitted.